### PR TITLE
Skill set and about me sections

### PR DESCRIPTION
**Pull request by yosaddis**
This feature are added
- my works section is added
- about me section is added

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -59,9 +59,162 @@ <h1>
 				</a>
 			</li>
 		</ul>
+		<a>
+			<img src="./img/icon/readmore.svg" alt="read more button">
+		</a>
+	</section>
+	<section id="my-works" >
+		<div class="works">
+			<h1>My Recent Works</h1>
```
Comment (joskalenda) [inline index.html]: - [x] The title here is supposed to be wrapped in an h2 -h6 tag and not in h1, following the HTML best practice, it is required to only have 1 h1 tag on a page, please change this.

File: index.html [hunk 2]
```diff
@@ -59,9 +59,162 @@ <h1>
...
+					<li>JavaScript</li>
+					<li>html</li>
+				</ul>
+				<button type="button" > See Project</button>
+			</div>
+		</div>
+	</section>
+	<section class="about-me-section" id="about-me">
+		<div class="container">
+			<h1>About me</h1>
```
Comment (joskalenda) [inline index.html]: - [ ] Same as here, please change this h1 into an h2 pr hx...6 tag

File: index.html [hunk 3]
```diff
@@ -59,9 +59,162 @@ <h1>
...
+				<div class="work-thumbnail">
+					<img  alt="">
+				</div>
+				<h2 class="card-title">Multi-Post Stories Gain+Glory</h2>
+				<ul class="card-languages" >
+					<li>Ruby on rails</li>
+					<li>css</li>
+					<li>JavaScript</li>
+					<li>html</li>
+				</ul>
```
Comment (joskalenda) [inline index.html:79]: - [ ] See the spaces around your technologies, and consider fixing them to match the design on the left

File: index.html [hunk 4]
```diff
@@ -59,9 +59,162 @@ <h1>
...
+						<img src="./img/icon/icon-languages.svg" alt="skill icon">
+					</div>
+					<h2 class="skill-name">Languages</h2>
+					<ul class="skill-list">
+						<li>JavaScript</li>
+						<li>Ruby</li>
+						<li>Html</li>
+						<li>CSS</li>
+					</ul>
+				</div>
```
Comment (joskalenda) [inline index.html:185]: - [ ] Take a look at the points I have highlighted and make the necessary changes to look like the original design, I think you need to add one more framework as per the design on the left

**Review by joskalenda — CHANGES_REQUESTED** [review]
| STATUS: REQUIRED CHANGES ♻️  |
|--------------------------------|
| Hi @yosaddis , |
|Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there! |
|  |

## Highlights

- [ ]... ✅ Great design
- [ ]... ✅ Good commit messages
- [ ]... ✅ Great read me file

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Kossi-stack — APPROVED** [review]
## Status: Approved ✔️

**Hi @yosaddis,**

You did a really great job! 💯 💯💯💯
Congratulations! 🎉 🎉 🎉 🎉 

128382030-38e7de91-f6f2-45cf-b54e-8068e0446e4c

You may close this Pull Request and merge the changes
Keep being awesome dev's and good luck with your next milestone!

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏 👏 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**